### ISSUE TITLE
API Key is written into log files

### ISSUE THREAD

**Issue by eickeh** [gophish/gophish#1416]
**What version of Gophish are you using?:** 
0.6.0

**Brief description of the issue:**
First of all, thanks for your work!

While debugging an issue with GoPhish, we've discovered that API keys are written to the logfile (or GoPhish console output), which is actually not that surprising since the admin backend sends the API key as a query string.
From my point of view, there would be two approaches to address this:
  * Filter the logfiles for sensitive data (i.e. the API key)
  * Move the API key to an authorization header.

Some short digging revealed, that there is already support for this in the middleware, and the python client already supports this. Further, there only seem to be two places (Bulk Upload in users.js and all remaining functionality in query() in gophish.js) in the frontend code which use the api_key as a query parameter.

Is there any specific reason preventing this from happening?
If there is interest, we would of course provide pull requests.

**What are you expecting to see happen? :**
No API keys appear in log files.

**What are you seeing happen?**
API keys end up in the logs.

Please provide any terminal output that may be relevant below:

```
123.456.789.1 - - [25/Feb/2019:09:22:12 +0100] "GET
/api/templates/?api_key=[ API key would be here]
HTTP/1.1" 200 853 ...
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Log into GoPhish
 * Step 2 - Click on Templates
 * Step 3 - View log output

**Comment by jordan-wright** [comment]
Hey there! There’s no reason not to have this in the frontend. I just haven’t had time to knock it out.

Pull requests would be very welcome!

**Comment by jordan-wright** [comment]
Closed via #1439
